examples/featherwing_tft24_simpletest.py

```diff
@@ -13,11 +13,15 @@
 tft_featherwing = tft_featherwing_24.TFTFeatherWing24()
 
 try:
-    f = open("/sd/tft_featherwing.txt", "w")
+    f = open(  # pylint: disable=unspecified-encoding,consider-using-with
```

Comment:
Why not use `with` as the lint suggests? It'd be:
```
with open("/sd/tft_featherwing.txt", "w") as f:
    f.write("Blinka\nBlackberry G10 Keyboard")
```

Reply:
@tannewt Ah yeah you're right. For some reason I thought it was opening 2 files at the same time.